committee-steering/governance/sig-governance.md

```diff
@@ -117,13 +116,21 @@ curation from other SIG participants
 
 ### Tech Lead
```

Comment:
do you also want to extend the document contributors/chairs-and-techleads/technical-lead.md as in https://github.com/kubernetes/community/pull/7095/files?

Reply:
We can update that one out of band 👍 it's more general best practices.